crates/democracy/src/benchmarking.rs

```diff
@@ -42,7 +42,7 @@ fn add_referendum<T: Config>(n: u32) -> Result<ReferendumIndex, &'static str> {
     let proposal_hash: T::Hash = T::Hashing::hash_of(&n);
     let vote_threshold = VoteThreshold::SimpleMajority;
 
-    Democracy::<T>::inject_referendum(T::LaunchPeriod::get(), proposal_hash, vote_threshold, 0u32.into());
+    Democracy::<T>::inject_referendum(T::VotingPeriod::get(), proposal_hash, vote_threshold, 0u32.into());
```

Comment:
Is that actually the voting period? I though the launch period was basically the period where people could second a proposal before it is a referendum and afterward it becomes a referendum and people can vote on it.

Reply:
The launch period was the interval at which proposals would get automatically tabled. The `VotingPeriod` is actually the right argument here, since it's the block number at which the referendum is supposed to end. But anyway that doesn't really matter here - it's only benchmarking code, and the actual value doesn't impact the weight.